rosdep/python.yaml

```
@@ -2668,6 +2668,11 @@ python-mock:
     xenial_python3: [python3-mock]
     yakkety: [python-mock]
     zesty: [python-mock]
+python-monotonic:
+  arch: [python-monotonic]
+  debian: [python-monotonic]
+  fedora: [python-monotonic]
```

```suggestion
  nixos: [pythonPackages.monotonic]
```

Based on the discussion I propose this Nix package addition.

---

This helps :)
Didn't have access to a pc

---

This suggestion accidentally removed the Fedora entry.